src/main.js

```diff
@@ -61,10 +65,22 @@ window.addEventListener("DOMContentLoaded", () => {
     // Navbar
     initializeNavbar();
 
+    // Footer FormIO Action
+    // Note: This is added here, as there is an issue with breadcrumbShorten() function.
+    //       Will move this once that issue is fixed.
+    const footerFormio = document.getElementById("qg-feedback-toggle");
```

Comment:
could wrap this in a function like they have done with initBreadcrumb/navbar etc so its self contained.